projects/assets/src/translations/translation-chunks-config.ts

```diff
@@ -75,4 +75,47 @@ export const translationChunksConfig: TranslationChunksConfig = {
     'register',
     'checkoutLogin',
   ],
+
+  organization: [
```

Comment:
Shouldn't those chunks be defined in my-account lib?

Reply:
Good catch. It should be dropped! 

It's already defined in `feature-libs\my-account\organization\assets\translations\translations.ts`. It's not the same as this list, but this one seems outdated.

Reply:
thanks guys!